build/flaws/broken-links.ts

```diff
@@ -234,6 +234,7 @@ export function getBrokenLinksFlaws(
     } else if (
       href.startsWith("https://developer.mozilla.org/") &&
       !href.startsWith("https://developer.mozilla.org/en-US/curriculum/") &&
+      !href.startsWith("https://developer.mozilla.org/en-US/observatory") &&
```

Comment:
This is the only one without a trailing slash. Is that right?

Reply:
@bsmth That's correct. We'll probably unify this in the next months, but not yet.